Adds steps for keeping product owner informed about deploy. (#15)
emory-libraries/aspace-deployment: README.md

### Deployment Process

#### Pre-deployment

> 1. Create a new release for [aspace_static_plugins](https://github.com/emory-libraries/aspace_static_plugins) if the latest release does not contain the most up to date changes, and ensure the new release has a tag number that follows semantic versioning. You can use the following [instructions](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) to create a new release. Use the github feature to automatically draft the release notes. When those post to the #archivesspace-notify slack channel, copy them and post to the #archivesspace channel, tagging the product owner.

Reviewer: ```suggestion
1. Create a new release for [aspace_static_plugins](https://github.com/emory-libraries/aspace_static_plugins) if the latest release does not contain the most up to date changes, and ensure the new release has a tag number that follows semantic versioning. You can use the following [instructions](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) to create a new release. Use the github feature to automatically draft the release notes. Link to the release notes in the #archivesspace channel.
```
